src/js/components/landing/LandingContent.jsx

```diff
@@ -26,24 +26,30 @@ export default class LandingContent extends React.Component {
     constructor(props) {
         super(props);
 
+        this.didUnmount = false;
```

Comment:
What is the point of this variable? You set it to false here and false in `componentDidMount` and don't do anything else with it.